config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'orders/index'
+  get 'orders/show'
+  get 'orders/new'
+  get 'orders/edit'
+  get 'orders/create'
+  get 'orders/update'
+  get 'orders/destroy'
```

Comment:
What are these for? `resources :orders` is already declared below, and it doesn't really make sense to use `get` for create/update/destroy